classes/hotel.js

```diff
@@ -1,8 +1,13 @@
 class Hotel {
-  constructor(name, rooms, bookings) {
+  constructor(name, rooms, bookings, allGuests) {
     this.name = name;
     this.rooms = rooms;
-    this.bookings = bookings;    
+    this.bookings = bookings;
+    this.guests = allGuests;   
```

Comment:
I might need to add this to my hotel class as well

Reply:
It was my hacky way of accessing the users after loading, but it works!